test/cpp/client/client_channel_stress_test.cc

```diff
@@ -133,6 +134,7 @@ class BalancerServiceImpl : public LoadBalancer::Service {
 class ClientChannelStressTest {
  public:
   void Run() {
+    EnableDefaultHealthCheckService(true);
```

Comment:
Consider having a test argument that sets whether or not to use the default health checker so that this change keeps the existing test scenarios and adds new ones rather than modifying the existing ones.

Reply:
I did consider that, but it didn't seem worth doing.  This only enables the health check service on the server side, and its presence doesn't actually affect the operation of tests that are not actually doing health checking.

The array on line 317 below shows the different types of channels we are now testing with on the client side, some of which do health checking and others of which do not.

Reply:
Sounds good. As long as both cases are well-tested.